src/main/java/org/mdbenefits/app/submission/actions/CheckExpeditedSnapEligibility.java

```diff
@@ -33,9 +33,10 @@ public void run(Submission submission) {
             BigDecimal rentAmount = new BigDecimal(rentString).setScale(2, RoundingMode.HALF_UP);
 
             boolean isMigrantOrSeasonalFarmWorker = inputData.get("migrantOrSeasonalFarmWorkerInd").toString().equals("true");
-            boolean hasUtilities =
-                    inputData.containsKey("householdUtilitiesExpenses[]") && !inputData.get("householdUtilitiesExpenses[]")
-                            .toString().equals("None");
+            // TODO: Update this logic with household expenses that are considered utilities
```

Comment:
@bseeger - this is how I added the TODO in the code. I think updating the logic is outside of the scope of this project. What do you think of this approach?

Reply:
I think this is fine.